perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -20,8 +20,18 @@
 import logging
 from perfkitbenchmarker import configs
 from perfkitbenchmarker import sample
+from perfkitbenchmarker import flags
+
 import re
 
+flags.DEFINE_enum('ping_net_interface_type', 'internal_ip',
```

Comment:
You're absolutely right, --ip_addresses param will cover the requirement to ping on the external interface. I'll remove the 'ping_net_interface_type' flags.